### DISCUSSION TITLE
Dropdown error in Action fields

### DISCUSSION THREAD

**Discussion by kathrynvillanueva** [spatie/nova-tags-field#175] [Q&A]
Dropdown not showing up. Options were fetched as I see in the dev tools network but an error appears in console. 
![image](https://user-images.githubusercontent.com/61833715/214644926-18d1ad9e-b4d1-4c22-aa1b-2d26a6bdeac5.png)

The tag field works perfectly fine in Resources but not in Actions.

**Comment by frankmichel** [comment]
@freekmurze I have created a PR that fixes this issue at https://github.com/spatie/nova-tags-field/pull/181

**Comment by frankmichel** [comment]
@freekmurze Unfortunately after fixing this, the saving does not work from Actions. 

The `$model` value in the `fillAttributeFromRequest` method in `Tags.php` is resolved to `Laravel\Nova\Support\Fluent` when saving tags through a Nova action.

This differs from saving tags in resources. There, the correct model is being resolved.

Since the model cannot be resolved correctly, the following error is thrown:
`Call to undefined method Laravel\\Nova\\Support\\Fluent::saved()`

Unfortunately I was not able to fix this.
